finder-app/finder-test.sh

```diff
@@ -2,6 +2,9 @@
 # Tester script for assignment 1 and assignment 2
 # Author: Siddhant Jajoo
 
+make clean
+make
+
```

Comment:
@samu7988 think on why did you add make clean and make here? don't you think you were expected to add them at some point later?